components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/AuthenticationRequest.java

```diff
@@ -57,19 +57,17 @@ public class AuthenticationRequest implements Serializable
 
     private static final Log log = LogFactory.getLog(AuthenticationRequest.class);
 
-    protected Map<String, Object> attributes = new HashMap<>();
-    protected Map<String, String> headers = new HashMap<>();
-    protected Map<CookieKey, Cookie> cookies = new HashMap<>();
```

Comment:
Shall we remove the corresponding `cookies` hash map in the `AuthenticationRequestBuilder` inner class as well?